portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,6 +31,18 @@
 import com.google.appengine.api.images.ServingUrlOptions;
 import com.google.gson.*;
 import com.google.gson.annotations.*;
+import com.google.cloud.vision.v1.AnnotateImageRequest;
+import com.google.cloud.vision.v1.AnnotateImageResponse;
+import com.google.cloud.vision.v1.BatchAnnotateImagesResponse;
+import com.google.cloud.vision.v1.EntityAnnotation;
+import com.google.cloud.vision.v1.Feature;
+import com.google.cloud.vision.v1.Image;
+import com.google.cloud.vision.v1.ImageAnnotatorClient;
+import com.google.cloud.vision.v1.LocalizedObjectAnnotation;
+import com.google.cloud.vision.v1.NormalizedVertex;
+import com.google.cloud.vision.v1.*;
```

Comment:
Generally we prefer not importing .* and list everything you want to import individually.